src/components/steps/Steps.vue

```diff
@@ -199,7 +199,7 @@ export default {
         * Change the active step and emit change event.
         */
         changeStep(newIndex) {
-            if (this.activeStep === newIndex) return
+            if (this.activeStep === newIndex || this.stepItems[newIndex]) return
```

Comment:
I'm not sure this would keep the correct behavior...

Reply:
I'm also not 100% sure.
The use case is: Having the steps inside pagination, when you go to page two, seems that the stepsArray is empty and "this.stepItems[newIndex].activate()" will fail.
It seems save to escape if there are no stepsItems.

Reply:
But right now, the function exit when the newindex exists in the stepItems. So the code below would never run.

The condition to return should probably be something like `newIndex < this.stepItems.length` instead. So the function will return early only if the new index is not part of the step items.

Reply:
You're totally right **newIndex** can never be bigger than the total index.
I've changed the PR and also added error message.
Thanks